SimpleClassicThemeTaskbar/Forms/CrashForm.cs

```diff
@@ -0,0 +1,143 @@
+    {
+        private readonly Exception exception;
+
+        public CrashForm()
+        {
+            InitializeComponent();
+        }
+
+        public CrashForm(Exception ex) : this()
+        {
+            exception = ex;
+        }
+
+        private void BackgroundWorker_DoWork(object sender, DoWorkEventArgs e)
+        {
+            if (exception == null)
+            {
+                MessageBox.Show(
+                    this,
+                    "No report could be sent because the application didn't provide an exception.",
+                    "Report failed to send",
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Warning);
+            }
+            else
+            {
+                using (SentrySdk.Init(o =>
+                {
+                    o.Dsn = "https://eadebff4c83e42e4955d85403ff0cc7c@o925637.ingest.sentry.io/5874762";
+                    o.MaxBreadcrumbs = 50;
+#if DEBUG
+                    o.Debug = true;
+#endif
+                }))
+                {
+                    SentrySdk.ConfigureScope(scope =>
+                    {
+                        if (submitLogCheckBox.Checked)
+                        {
+                            scope.AddAttachment(Logger.FilePath, AttachmentType.Default);
+                        }
+
+                        SentrySdk.CaptureException(exception);
+                    });
+                }
+            }
+        }
+
+        private void BackgroundWorker_RunWorkerCompleted(object sender, RunWorkerCompletedEventArgs e)
+        {
+            if (e.Error == null)
+            {
+                MessageBox.Show(
+                    this,
+                    "Report has been sent successfully.",
+                    "Report sent",
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Information);
+            }
+            else
+            {
+                MessageBox.Show(
+                    this,
+                    "We failed to send the error report. Please check the application logs and explain this occurrance over at GitHub.",
+                    "This is embarrassing...",
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Warning);
+            }
+
+            Close();
+        }
+
+        private void CrashForm_FormClosing(object sender, FormClosingEventArgs e)
+        {
+            if (backgroundWorker.IsBusy)
+            {
+                e.Cancel = true;
+            }
+            else
+            {
+                Environment.Exit(1);
+            }
+        }
+
+        private void DescriptionLinkLabel_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            MessageBox.Show(this,
+                @"The currently known data collected by error reports sent in debug builds is:
+
+• Current application log
+    • OS version
+    • SCT and application version
+    • Potentially currently running depending on debug output
```

Comment:
This is vague

Reply:
What can I do to improve it?

Reply:
'Depending on the problem, this might also include information about open windows'

Reply:
I notice my head skipped a word again when writing this, I see.